docs/modules/advanced_speed_servo_info.rst

```diff
@@ -42,6 +42,7 @@ Supported Tutorials
         * :ref:`hobby_fc_tutorial`
         * :ref:`hobby_calibration_tutorial`
         * :ref:`dronecan_fc_tutorial`
+        * :ref:`dronecan_firmware_upgrade`
```

Comment:
this is applicable to all of our advanced speed modules, so add it to the applicable tutorials list.